Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Fix FTBFS (test failure) with GCC 6.1
 Hex float constants are part of C++17 (and only GNU extensions in C++11)
 and GCC 6.1 errors out when src/mongo/base/parse_number_test.cpp is
 compiled using -std=c++11.
Last-Update: 2016-11-11
Forwarded: no (legacy)
--- a/src/mongo/base/parse_number_test.cpp
+++ b/src/mongo/base/parse_number_test.cpp
@@ -279,7 +279,7 @@
 
     ASSERT_PARSES(double, "0xff", 0xff);
     ASSERT_PARSES(double, "-0xff", -0xff);
-    ASSERT_PARSES(double, "0xabcab.defdefP-10", 0xabcab.defdefP-10);
+    //ASSERT_PARSES(double, "0xabcab.defdefP-10", 0xabcab.defdefP-10);
 #endif
 }
 
